CDP4-CE.sln.DotSettings

```diff
@@ -255,6 +255,6 @@
 	<s:Boolean x:Key="/Default/Environment/SettingsMigration/IsMigratorApplied/=JetBrains_002EReSharper_002EPsi_002ECSharp_002ECodeStyle_002ESettingsUpgrade_002EMigrateBlankLinesAroundFieldToBlankLinesAroundProperty/@EntryIndexedValue">True</s:Boolean>
-	<s:String x:Key="/Default/FilterSettingsManager/CoverageFilterXml/@EntryValue">&lt;data&gt;&lt;IncludeFilters /&gt;&lt;ExcludeFilters&gt;&lt;Filter ModuleMask="CDP4Grapher" ModuleVersionMask="*" ClassMask="CDP4Grapher.Views.*" FunctionMask="*" IsEnabled="True" /&gt;&lt;Filter ModuleMask="CDP4Grapher" ModuleVersionMask="*" ClassMask="CDP4Grapher.Behaviors.GrapherOrgChartBehavior" FunctionMask="ExportGraph" IsEnabled="True" /&gt;&lt;Filter ModuleMask="CDP4Grapher" ModuleVersionMask="*" ClassMask="CDP4Grapher.Utilities.GrapherSaveFileDialog" FunctionMask="*" IsEnabled="True" /&gt;&lt;Filter ModuleMask="CDP4Grapher" ModuleVersionMask="*" ClassMask="CDP4Grapher.Behaviors.GrapherOrgChartBehavior" FunctionMask="Loaded" IsEnabled="True" /&gt;&lt;Filter ModuleMask="CDP4Grapher" ModuleVersionMask="*" ClassMask="CDP4Grapher.Behaviors.GrapherOrgChartBehavior" FunctionMask="OnDataContextChanged" IsEnabled="True" /&gt;&lt;/ExcludeFilters&gt;&lt;/data&gt;</s:String>
+	<s:String x:Key="/Default/FilterSettingsManager/CoverageFilterXml/@EntryValue">&lt;data&gt;&lt;IncludeFilters /&gt;&lt;ExcludeFilters&gt;&lt;Filter ModuleMask="CDP4Grapher" ModuleVersionMask="*" ClassMask="CDP4Grapher.Views.*" FunctionMask="*" IsEnabled="True" /&gt;&lt;Filter ModuleMask="CDP4Grapher" ModuleVersionMask="*" ClassMask="CDP4Grapher.Behaviors.GrapherOrgChartBehavior" FunctionMask="ExportGraph" IsEnabled="True" /&gt;&lt;Filter ModuleMask="CDP4Grapher" ModuleVersionMask="*" ClassMask="CDP4Grapher.Utilities.GrapherSaveFileDialog" FunctionMask="*" IsEnabled="True" /&gt;&lt;Filter ModuleMask="CDP4Grapher" ModuleVersionMask="*" ClassMask="CDP4Grapher.Behaviors.GrapherOrgChartBehavior" FunctionMask="Loaded" IsEnabled="True" /&gt;&lt;Filter ModuleMask="CDP4Grapher" ModuleVersionMask="*" ClassMask="CDP4Grapher.Behaviors.GrapherOrgChartBehavior" FunctionMask="OnDataContextChanged" IsEnabled="True" /&gt;&lt;Filter ModuleMask="CDP4Composition" ModuleVersionMask="*" ClassMask="CDP4Composition.Behaviors.PluginUpdateInstallerBehavior" FunctionMask="OnClosing" IsEnabled="True" /&gt;&lt;Filter ModuleMask="CDP4Composition" ModuleVersionMask="*" ClassMask="CDP4Composition.Behaviors.PluginUpdateInstallerBehavior" FunctionMask="OnAttached" IsEnabled="True" /&gt;&lt;Filter ModuleMask="CDP4Composition" ModuleVersionMask="*" ClassMask="CDP4Composition.Behaviors.PluginUpdateInstallerBehavior" FunctionMask="OnDetaching" IsEnabled="True" /&gt;&lt;Filter ModuleMask="CDP4Composition" ModuleVersionMask="*" ClassMask="CDP4Composition.Behaviors.PluginUpdateInstallerBehavior" FunctionMask="OnDataContextChanged" IsEnabled="True" /&gt;&lt;/ExcludeFilters&gt;&lt;/data&gt;</s:String>
```

Comment:
I think actually both these lines should not be there. I think this is user coverage masking, and should not be part of version control. So revert and then remove line 258 alltogether.